deployments/deploy.js

```diff
@@ -4,11 +4,12 @@ async function main() {
     const UsersInfo = await ethers.getContractFactory("UsersInfo");
     const user_info = await UsersInfo.deploy();
```

Comment:
The 'console.log("Nice")' statement does not provide any useful information and should be removed.